aspnetcore/signalr/hubcontext.md

```diff
@@ -0,0 +1,56 @@
+---
+title: SignalR HubContext
+author: rachelappel
+description: Overview of the use of SignalR's HubContext
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 6/12/18
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/HubContext
+---
+
+# Sending Messages Outside of you Hub Class
+
+By [Mikael Mengistu](https://github.com/mikaelm12)
+
+The SignalR Hub is the core abstraction for sending messages to clients connectied to our SignalR server. You aren't limited to sending messages to clients from within a Hub though. Here we'll look at how to access a SignalR HubContext to send notification to clients from outside of a hub.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/signalr/hubcontext/sample/) [(how to download)](xref:tutorials/index#how-to-download-a-sample)
+
+## Getting and Using the HubContext
+
+In ASP.NET Core SignalR you can get access to an instance of HubContext via dependency injection. You can get an instance of `HubContext<YourHubType>` in an MVC Controller or middleware this way. In previous versions of SignalR you would do this by explicitly asking for an instance of HubContext via a ConnectionManager but that has been replaced in favor of our dependency injection approach.
```

Comment:
HubContext -> `IHubContext`

Reply:
You can get an instance of `HubContext<YourHubType>` in an MVC Controller or middleware this way.

->

You can inject an instance of `IHubContext<YourHubType>` into a Controller, Middleware or other DI service and use it to send messages to clients.

Reply:
In previous versions of SignalR you would do this by explicitly asking for an instance of HubContext via a ConnectionManager but that has been replaced in favor of our dependency injection approach.

-> move to a Note call-out (see some of @BrennanConroy 's recent PRs for the syntax)

This differs from ASP.NET SignalR which used `GlobalHost` to provide access to the `IHubContext` because there was no built-in dependency injection system.